client/my-sites/plugins/index.js

```diff
@@ -76,6 +76,14 @@ module.exports = function() {
 			pluginsController.plugin
 		);
 
+		if ( config.isEnabled( 'automated-transfer' ) ) {
+			page( '/plugins/:plugin/:site_id/eligibility',
```

Comment:
I'm pausing here to reflect on the URL structure:

 - do we want to expose `eligibility` in the URL? probably?
 - do we want to tie this to `/plugins`? maybe
 - do we want it instead to be at `/transfers/eligibility`? maybe not?

this would be a good thing for a design review.